learn/configuration/synonyms.md

```diff
@@ -2,8 +2,15 @@
 
 If multiple words have an equivalent meaning in your dataset, you can [create a list of synonyms](/reference/api/synonyms.md#update-synonyms). This will make your search results more relevant.
 
-In general, **a search on a word will return the same results as a search on any of its synonyms**.
-There is one exception to this rule, [detailed below](#multi-word-phrases).
+Words set as synonyms won't always return the same results. With the default settings, the `movies` dataset should return 547 results for `great` and 66 for `fantastic`. Let's set them as synonyms:
+
+<CodeSamples id="synonyms_guide_1" />
+
+With the new settings, searching for `great` returns 595 results and `fantastic` returns 423 results. This is due to various factors like [typos](/learn/configuration/typo_tolerance.md#minwordsizefortypos) and [splitting the query](/learn/advanced/concat.md#split-queries) to find relevant documents. The search for `great` will allow only one typo and take into account all variations of `great`, e.g., `creatures`, `greatest`, etc. along with `fantastic`.
```

Comment:
As I mentioned in my previous comment, here's my suggestion: we separate the examples of typos and word variations so it's clear what we're referring to. I also shortened `creatures` to `create` hoping that less letters might make it easier for users to identify the typo.

```suggestion
With the new settings, searching for `great` returns 595 results and `fantastic` returns 423 results. This is due to various factors like [typos](/learn/configuration/typo_tolerance.md#minwordsizefortypos) and [splitting the query](/learn/advanced/concat.md#split-queries) to find relevant documents. The search for `great` will allow only one typo (e.g. `create`) and take into account all variations of `great` (e.g. `greatest`) along with `fantastic`.
```

We could also wrap the matching term with `<em>`, if you think it's necessary.